doc/devdocs/modules/launcher/plugins/registry.md

```diff
@@ -90,12 +90,13 @@ The plugin use only these interfaces (all inside the `Main.cs`):
 
 #### Packages
 
-| Package                                                                               | Version |
-| ------------------------------------------------------------------------------------- | ------- |
-| [`Microsoft.CodeAnalysis.FxCopAnalyzers`](https://github.com/dotnet/roslyn-analyzers) | 3.3.0   |
-| [`StyleCop.Analyzers`](https://github.com/DotNetAnalyzers/StyleCopAnalyzers)          | 1.1.118 |
+<!-- fxcop is deprecated -->
+| Package                                                                             | Version |
+| ----------------------------------------------------------------------------------- | ------- |
```

Comment:
this may be dated at this point.  I think @CleanCodeDeveloper migrated us off if memory serves

Reply:
fxcop is no longer a thing. There are a few comments and suppressions regarding fxcop left which I want to clean up (I have no spare time at the moment because parenting is hard).

Reply:
I thought so. Just kept it on the safe side here&now. I suppose simply deleting this line is enough (?)